### PR TITLE
Feat: Integration test cases for configlet_build_config

### PR DESCRIPTION
**Pull request by ghost**
## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.<role-name>`

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by KonikaChaurasiya-GSLab** [comment]
Hi @Shivani-chourasiya - Below are the review comments;

1. Inventory file location change should not affect existing testcases
2. ansible_collections/arista/avd/tests/integration/inventory : 
    Harcoded you ansible_python_interpreter in the file
   ansible_python_interpreter="/Users/schourasiya-ext/ansible1/bin/python" 
3. Is there any need of empty file main.yaml in meta directory
4. Use meaningful variable name in defaults/main.yml
   var: "/../../../../../../../../inventory/intended/configs"
5. Do not commit the output folder
6. Check all the validation should be pass, after commiting the code in the PR.

**Comment by Shivani-chourasiya** [comment]
> Hi @Shivani-chourasiya - Below are the review comments;
> 
> 1. Inventory file location change should not affect existing testcases
> 2. ansible_collections/arista/avd/tests/integration/inventory :
>    Harcoded you ansible_python_interpreter in the file
>    ansible_python_interpreter="/Users/schourasiya-ext/ansible1/bin/python"
> 3. Is there any need of empty file main.yaml in meta directory
> 4. Use meaningful variable name in defaults/main.yml
>    var: "/../../../../../../../../inventory/intended/configs"
> 5. Do not commit the output folder
> 6. Check all the validation should be pass, after commiting the code in the PR.

**Comment by Shivani-chourasiya** [comment]
Hi @KonikaChaurasiya-GSLab ,
1. Inventory location is updated as per discussion, need to change in unit testcases
2. ansible_collections/arista/avd/tests/integration/inventory , this is generated by default, will remove this while committing.
3.  removing main.yml from meta folder
4. changed the variable name to cofiglet_path
5. Removing the output folder

File: ansible_collections/arista/avd/tests/integration/targets/configlet_build_config/tests/test_without_configletDir.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- name: without configlet_dir
```
Comment (manuwelakanade) [inline ansible_collections/arista/avd/tests/integration/targets/configlet_build_config/tests/test_without_configletDir.yml]: Change file name to configlet_dir

**Review by manuwelakanade — APPROVED** [review]

**Pull request merged**